### ISSUE TITLE
Toolbar: avt check failing with accessibility violations

### ISSUE THREAD

**Issue by amal-k-joy** [carbon-design-system/ibm-products#6160]
avt checks are failing for any new PR raised and throwing accessibility violations from Toolbar component.

> Message: Undersized target "span" does not have sufficient spacing of 12 CSS pixels from another target "button"
>       Level: violation
>       XPath: /html[1]/body[1]/div[5]/div[1]/div[1]/div[5]/span[1]/div[1]/button[1]/span[1]
>       Snippet: <span class="c4p--toolbar__button__caret">
>       Help: https://able.ibm.com/rules/archives/2024.10.01/doc/en-US/target_spacing_sufficient.html#%7B%22message%22%3A%22Undersized%20target%20%5C%22span%5C%22%20does%20not%20have%20sufficient%20spacing%20of%2012%20CSS%20pixels%20from%20another%20target%20%5C%22button%5C%22%22%2C%22snippet%22%3A%22%3Cspan%20class%3D%5C%22c4p--toolbar__button__caret%5C%22%3E%22%2C%22value%22%3A%5B%22VIOLATION%22%2C%22FAIL%22%5D%2C%22reasonId%22%3A%22violation_spacing%22%2C%22ruleId%22%3A%22target_spacing_sufficient%22%2C%22msgArgs%22%3A%5B%22span%22%2C%22button%22%5D%7D
